### ISSUE TITLE
함수 이름 및 기능과 사용 방법

### ISSUE THREAD

**Issue by haram22** [haram22/2022OSS_team_KH#3]

**Comment by haram22** [comment]
Update 기능의 함수 이름은 update
Delete 기능의 함수 이름은 delete_r

어떠세요 ~?

**Comment by majink** [comment]
Create 함수의 이름 : Reservation 
Read 함수의 이름 : Readfile

**Comment by majink** [comment]
혹시 괜찮으시면 이름 앞 대문자로 하는 건 어떠신 가요 ?

**Comment by haram22** [comment]
좋아요 ! 그럼 Update, Delete_r로 할게요

**Comment by majink** [comment]
메뉴를 보여주는 메뉴함수는 Menu로 하겠습니다!

**Comment by haram22** [comment]
CRUD에서 R의 이름을 Readfile로 하려고 보니, 나중에 파일을 읽어오는 함수를 만들게 된다면 헷갈릴 것 같아서 PrintList라고 이름을 바꿨으니 확인 부탁드려요 !

**Comment by majink** [comment]
네 ! 확인했습니다. 그리고 파일을 읽어오는 건 ReadData로 그리고 파일을 저장하는 건 SaveData로 하면 좋을 것 같아요!

**Comment by haram22** [comment]
selectNo 함수는 어떤 함수인가요?
제가 번호만 입력받고 넘겨주는 함수로 input()이라는 함수를 만들었는데, 혹시 같은 기능일까요?

**Comment by haram22** [comment]
아 혹시 selectNo에서 예약자 이름을 입력받는 이유를 알 수 있을까요!?

**Comment by majink** [comment]
아 미리 대답 못해서 죄송합니다! 수정 함수와 삭제 함수에서 몇번째의 특정 정보를 수정이나 삭제를 원하는 지 먼저 묻고 그 특정 정보의 순서를 수정함수와 삭제함수에 넘겨주어야한다고 생각했기 때문에 만들었었습니다!

**Comment by majink** [comment]
그래서 selectNo의 return 값이 int 값인 이유가 특성 순서를 나타낸다는 것을 알 수 있고, 삭제 함수와 수정 함수의 파라미터에 integer 값이 포함된 이유인 것 같습니다(여기서 count는 전체가 아니라 특성 순서를 나타내는 파라미터라고 생각하고 짰습니다!)
